src/task/style.css

```diff
@@ -258,3 +258,23 @@ footer {
   color: rgb(255, 92, 92);
   font-size: 0.75rem;
 }
+
+#assigneeAvatar {
+  background: right / 42px;
```

Comment:
Does this need to be wrapped around **calc()** ?

Reply:
no, actually I added background-position and background-size property in this, and this is how the syntax is, if we add position and size, have to add slash ( / ), otherwise it doesn't work

Reply:
That was help, Thank you